### DISCUSSION TITLE
Nextjs distributed tracing using opentelemetry or AWS xray

### DISCUSSION THREAD

**Discussion by tkumark** [vercel/next.js#62125] [Help]
### Summary

Has anyone implemented distributed tracing for Nextjs using opentelemetry or AWS xray in production. From the Nextjs site Opentelemetry is still expremental has anyone used it production. Any other suggestion for distributed tracing for Nextjs? Any idea when the opentelemetry implemention will be out of experimental

**Comment by ghost** [comment]
You can [check this page](https://nextjs.org/docs/app/building-your-application/optimizing/open-telemetry), which explains how to configure OpenTelemetry for the application.
